packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/NotOwnedSandboxInfo/Summary.tsx

```diff
@@ -164,13 +171,27 @@ export const Summary = () => {
 
         {!author && currentSandbox.git ? (
           <Stack as="section" direction="vertical" gap={4} paddingX={2}>
-            <Text variant="muted" size={3}>
-              This sandbox is in sync with{' '}
-              <Text weight="bold">{currentSandbox.git.branch}</Text> on GitHub.
-              You have to fork to make changes
-            </Text>
+            {!isPro && isUnlistedOrPrivate ? (
+              <Text variant="muted" size={3}>
+                This is {privacy === 1 ? 'a private' : 'an unlisted'} sandbox.{' '}
```

Comment:
Do we need whitespace after the conditional? Looks like the words are not separated:

![image](https://user-images.githubusercontent.com/24959348/216325372-a03041b6-2dc1-4519-b42e-8b5a28dbe1d6.png)

Reply:
Huh it's weird that it shows up like this, I'll look into it.

Reply:
Has been fixed in the meantime, with the new text!